.ci/Jenkinsfile

```diff
@@ -249,6 +249,7 @@ pipeline {
               script {
                 def suitesParam = params.runTestsSuites
                 def existingSuites = readYaml(file: "${env.WORKSPACE}/${env.BASE_DIR}/${TEST_MATRIX_FILE}")
+                log(level: 'DEBUG', text: "${existingSuites}")
```

Comment:
Will revert this. I'm seeing that any build from the packaging job is adding all the suites, not honouring the matrix file.